root/hist/chain/CMakeLists.txt

```diff
@@ -1 +1,2 @@
+include_directories(${GTEST_INCLUDE_DIR})
```

Comment:
Why does this need to be repeated in each CMakeList.txt?

Reply:
Not in every CMakeLists.txt, but only where is used gtest, before gtest includes were inside source tree and was easily located, now we need to add them to includes, no?

Reply:
Do you want to move it in ROOTTEST_ADD_UNITTEST_DIR macro?

Reply:
Well 'sort of'.  The include_directories(${GTEST_INCLUDE_DIR}) should be done at the global level in a similar place that induce the definition of ROOTTEST_ADD_UNITTEST_DIR  to be available (without any explicit include) in each CMakeList.txt.